### ISSUE TITLE
Could not find enough ServerJSDefine

### ISSUE THREAD

**Issue by mclaughlinconnor** [carpedm20/fbchat#580]
## Description of the problem
Logging in fails. These may be two separate issues. I can log in fine with `1.9.6` and the session file was generated with it.

## Code to reproduce
When logging in with a saved session:
```py
cookies = load_cookies()
session = fbchat.Session.from_cookies(cookies)
```
## Traceback
```
Traceback (most recent call last):
  File "/home/connor/Python/History Book/listen.py", line 115, in <module>
    session = fbchat.Session.from_cookies(cookies)
  File "/home/connor/Python/History Book/.venv/lib/python3.7/site-packages/fbchat/_session.py", line 425, in from_cookies
    return cls._from_session(session=session)
  File "/home/connor/Python/History Book/.venv/lib/python3.7/site-packages/fbchat/_session.py", line 382, in _from_session
    define = parse_server_js_define(r.content.decode("utf-8"))
  File "/home/connor/Python/History Book/.venv/lib/python3.7/site-packages/fbchat/_session.py", line 36, in parse_server_js_define
    raise _exception.ParseError("Could not find enough ServerJSDefine", data=html)
fbchat.ParseError: Could not find enough ServerJSDefine. Please report this, along with the data below!
```
[data](https://gist.github.com/mclaughlinconnor/edf7ea00a5a4d3c41815d5007a1704e6)

## Code to reproduce
When logging in without a saved session:
```py
session = fbchat.Session.login(username, password, on_2fa_callback=lambda: input("2FA code: "))
```

## Traceback

```py
  File "/home/connor/Python/History Book/listen.py", line 119, in <module>
    session = fbchat.Session.login(username, password, on_2fa_callback=fa_handle)
  File "/home/connor/Python/History Book/.venv/lib/python3.7/site-packages/fbchat/_session.py", line 322, in login
    raise _exception.NotLoggedIn("Failed logging in: {}, {}".format(url, error))
fbchat.NotLoggedIn: Failed logging in: https://www.messenger.com/login/checkpoint_interstitial/?next=https%3A%2F%2Fwww.facebook.com%2Fcheckpoint%2Fstart%2F%3Fip%3D82.3 ...
```

## Environment information
- Python version: 3.7.7
- `fbchat` version: git `b1e438d`

**Comment by mclaughlinconnor** [comment]
Duplicate of #564

**Comment by madsmtm** [comment]
Hmm, actually not a duplicate - but the issue should be resolved by `v2.0.0a5`, in either 6141cc5a41320c9aaa03fb0721767284a713f0d7 or 8ac6dc4ae682cdc1cf26379f9737e788c6c0d269
